theme/theme.inc

```diff
@@ -234,11 +234,19 @@ function islandora_solr_metadata_query_fields($object, &$solr_fields) {
               // Will get stripped out later.
               return;
             }
-            $solr_query = format_string('!field:"!value"', array(
+            $solr_query = format_string('!field:!value', array(
               '!field' => $solr_field,
-              '!value' => urlencode(islandora_solr_replace_slashes(islandora_solr_lesser_escape($original_value))),
+              '!value' => islandora_solr_facet_escape($original_value),
+            ));
+            return t('<a href="@url">@val</a>', array(
```

Comment:
Is that referring to previous experimentation, appending to the path parameter itself in different ways?

[`l()`](https://api.drupal.org/api/drupal/includes!common.inc/function/l/7) effectively degrades to this (assuming links aren't being taken over in the theme):

```php
function l($text, $path, array $options = array()) {
  // [... figure out $use_theme and handle "active" class ...] 
  if ($use_theme) {
    return theme('link', array('text' => $text, 'path' => $path, 'options' => $options));
  }
  // The result of url() is a plain-text URL. Because we are using it here
  // in an HTML argument context, we need to encode it properly.
  return '<a href="' . check_plain(url($path, $options)) . '"' . drupal_attributes($options['attributes']) . '>' . ($options['html'] ? $text : check_plain($text)) . '</a>';
}
```

`@` in `t()`/`format_string()` == `check_plain()`